imblearn/over_sampling/tests/test_smote.py

```diff
@@ -140,30 +139,46 @@ def test_sample_borderline2():
     assert_array_equal(y_resampled, y_gt)
 
 
+@pytest.mark.filterwarnings('ignore:"kind" is deprecated in 0.4 and will be')
+@pytest.mark.filterwarnings('ignore:"svm_estimator" is deprecated in 0.4 and')
+@pytest.mark.filterwarnings('ignore:"out_step" is deprecated in 0.4 and')
+@pytest.mark.filterwarnings('ignore:"m_neighbors" is deprecated in 0.4 and')
 def test_sample_svm():
     kind = 'svm'
     smote = SMOTE(random_state=RND_SEED, kind=kind)
     X_resampled, y_resampled = smote.fit_sample(X, Y)
-    X_gt = np.array([[0.11622591, -0.0317206], [0.77481731, 0.60935141], [
-        1.25192108, -0.22367336
-    ], [0.53366841, -0.30312976], [1.52091956, -0.49283504], [
-        -0.28162401, -2.10400981
-    ], [0.83680821, 1.72827342], [0.3084254, 0.33299982], [
-        0.70472253, -0.73309052
-    ], [0.28893132, -0.38761769], [1.15514042, 0.0129463], [
-        0.88407872, 0.35454207
-    ], [1.31301027, -0.92648734], [-1.11515198, -0.93689695], [
-        -0.18410027, -0.45194484
-    ], [0.9281014, 0.53085498], [-0.14374509, 0.27370049],
-                     [-0.41635887, -0.38299653], [0.08711622, 0.93259929],
-                     [1.70580611, -0.11219234], [0.47436888, -0.2645749],
-                     [1.07844561, -0.19435291], [1.44015515, -1.30621303]])
-    y_gt = np.array(
-        [0, 1, 0, 0, 0, 1, 1, 1, 1, 1, 1, 0, 0, 1, 1, 1, 1, 0, 1, 0, 0, 0, 0])
+    X_gt = np.array([[0.11622591, -0.0317206],
+                     [0.77481731, 0.60935141],
+                     [1.25192108, -0.22367336],
+                     [0.53366841, -0.30312976],
+                     [1.52091956, -0.49283504],
+                     [-0.28162401, -2.10400981],
+                     [0.83680821, 1.72827342],
+                     [0.3084254, 0.33299982],
+                     [0.70472253, -0.73309052],
+                     [0.28893132, -0.38761769],
+                     [1.15514042, 0.0129463],
+                     [0.88407872, 0.35454207],
+                     [1.31301027, -0.92648734],
+                     [-1.11515198, -0.93689695],
+                     [-0.18410027, -0.45194484],
+                     [0.9281014, 0.53085498],
+                     [-0.14374509, 0.27370049],
+                     [-0.41635887, -0.38299653],
+                     [0.08711622, 0.93259929],
+                     [1.70580611, -0.11219234],
+                     [0.47436887, -0.2645749],
+                     [1.07844562, -0.19435291],
+                     [1.44228238, -1.31256615],
+                     [1.25636713, -1.04463226]])
+    y_gt = np.array([0, 1, 0, 0, 0, 1, 1, 1, 1, 1, 1, 0, 0, 1, 1, 1,
+                     1, 0, 1, 0, 0, 0, 0, 0])
     assert_allclose(X_resampled, X_gt, rtol=R_TOL)
     assert_array_equal(y_resampled, y_gt)
 
 
+@pytest.mark.filterwarnings('ignore:"kind" is deprecated in 0.4 and will be')
```

Comment:
It looks like all these filter warnings require some extra text.

Reply:
This is a regular expression. Only the beginning is useful (mainly the name of the parameter)